linter.py

```diff
@@ -24,7 +24,7 @@
 DEFAULT_CONFIG = {
```

Comment:
does the config file still work with this new installation and the submodule support? We should probably add that to the installation instructions...

Reply:
yes, this is still working; I'll add it to the documentation

Reply:
should the config file be renamed to `.linterconfig.yaml` (added dot in front) so it's hidden like other config files? @igilitschenski wdyt?

Reply:
Yeah, this makes sense.